dockerfiles/theia/docker/alpine/runtime-install-dependencies.dockerfile

```diff
@@ -5,4 +5,5 @@
 # Install curl and bash
 # Install ssh for cloning ssh-repositories
 # Install less for handling git diff properly
-RUN apk add --update --no-cache sudo git bzip2 which bash curl openssh openssh-keygen less
+# Install sshpass for handling passwordds for SSH keys
+RUN apk add --update --no-cache sudo git bzip2 which bash curl openssh openssh-keygen less sshpass
```

Comment:
Do we need the same dependency for ubi8 image? https://github.com/eclipse/che-theia/blob/3835a113e6c409d35c1282335f067cdc19aa87d3/dockerfiles/theia/docker/ubi8/runtime-install-dependencies.dockerfile

Reply:
done